scylla/src/cluster/metadata.rs

```diff
@@ -773,30 +777,62 @@ mod control_connection {
     #[derive(Clone)]
     pub(super) struct ControlConnection {
         conn: Arc<Connection>,
+        overriden_timeout: Option<Duration>,
     }
```

Comment:
Could you change the name to indicate that it is about server-side timeout?

Reply:
Very good idea.

Reply:
Done in a new commit.